pywr/solvers/cython_glpk.pyx

```diff
@@ -47,16 +47,27 @@ cdef class CythonGLPKSolver:
     cdef list aggregated
     cdef public object stats
 
+    # Internal representation of the basis for each scenario
+    cdef int[:, :] row_stat
+    cdef int[:, :] col_stat
+    cdef bint is_first_solve
+    cdef bint has_presolved
+    cdef bint use_presolve
```

Comment:
I didn't help much in our biggest model. So I wouldn't be too upset if this was removed or turned off by default. Currently we have no easy mechanism to pass arbitrary arguments to solvers.